images/ide/intellij/gradle/build.gradle

```diff
@@ -0,0 +1,43 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.google.gradle:osdetector-gradle-plugin:1.6.0'
+    }
+}
+
+apply plugin: 'application'
+apply plugin: 'com.google.osdetector'
+
+ext.platform = osdetector.os == 'osx' ? 'mac' : osdetector.os == 'windows' ? 'win' : osdetector.os
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile "org.openjfx:javafx-base:11:$platform"
```

Comment:
might be best to have 11 as a parameter, so that we can change to e.g. 11.0.1

Reply:
Do you mean that we define it as a macro for the whole docs?

Reply:
I was actually referring to e.g. ext.version = '11' in build.gradle. But it might be better as a macro so that is automatically changed in all docs